extensions/objects/models/objects.py

```diff
@@ -44,12 +44,15 @@ class BaseObject(object):
     def normalize(cls, raw):
         """Validates and normalizes a raw Python object.
 
+        Args:
+            raw: *. A normalized Python object to be normalized.
+
         Returns:
-          a normalized Python object describing the Object specified by this
-          class.
+          *. A normalized Python object describing the Object specified by this
```

Comment:
Here and below: the format of the normalized object should be known, right?

Reply:
This is a generic method checking and normalising the object based on the schema passed as second argument. [This](https://github.com/oppia/oppia/blob/5789c157aff939a5c0134ecc38cee48f11172570/schema_utils.py#L55) method in `schema_utils.py` demonstrates this. So, I think we can't specify the exact format.

Reply:
Oh yes, sorry, this one is in the base class, so that's fine. But the methods below would be for a specific object and would therefore be normalizing to a specific type, right?

E.g. for Boolean it would be

    bool. The normalized object (or False if the input is None or '').

Reply:
Done, PTAL!